# software_tools_for_synthetic_biology

### *Synthetic Biology Software*

[BindNet](https://github.com/fengshikun/BindNet): protein-ligand binding representation learning from fine-grained interactions

[BioNetGen](https://bionetgen.org/): structure-based modeling of biochemical reaction networks

[BioCRNplyer](https://github.com/BuildACell/bioCRNpyler): biomolecular chemical reaction network compiler 

[BioGRID](https://thebiogrid.org): large database of biological interactions modeled at the molecular level (proteins, genes, chemicals), not at the atomic bond level 

[BioNetGen](https://github.com/RuleWorld/bionetgen): rule based biochemical reaction network

[Cello](https://github.com/CIDARLAB/cello): genetic circuit design using logic gates 

[COPASI](https://copasi.org/): modeling biochemical reaction networks

[COBRA](https://opencobra.github.io/): whole-cell metabolic modeler that you can call as a package (*E. Coli*, etc) 

[DNA Chisel](https://github.com/Edinburgh-Genome-Foundry/DnaChisel/tree/master): codon optimization and solving sequence constraints 

[iBioSim](https://github.com/MyersResearchGroup/iBioSim): genetic circuit simulation and analysis 

[KBase](https://www.notion.so/Synthetic-Biology-1dc5b5e4de31802ea574ed4bb795ebc1?pvs=21): AWS for systems bio analysis, hosted by the Department of Energy

[PoSeiDon](https://github.com/rnajena/poseidon): a pipeline to detect significantly positive selected sites and possible recombination events in an alignment of multiple protein-coding sequences 

[PySB](https://pysb.org/): systems biology modeling in Python

[RecombineX](https://github.com/yjx1217/RecombineX): genome engineering toolkit 

[SBML](https://sbml.org/): standardized models for systems biology as a markup language, use it as a data format to describe synthetic biology code 

[SBOL canvas](https://sbolcanvas.org/): genetic circuit schematic building using SBOL 

[SnapGene](https://www.snapgene.com/): visual plasmid construction + simulation 

[SpliceVarDB](https://compbio.ccia.org.au/splicevardb/): variant database for splicing 

[TCellMatch](https://github.com/theislab/tcellmatch): collection of models to predict which antigens a T-cell receptor will bind, based on single-cell sequence data; test immune design before lab work  

[Uniprot](https://www.uniprot.org)/[PDB](https://www.rcsb.org): protein sequence + 3D structure databases

### *SynthBio Tools*

[Biometra TRobot](https://www.analytik-jena.com/products/life-science/pcr-qpcr-thermal-cycler/thermal-cycler-pcr/biometra-trobot-ii-series/): PCR machine 

[Digital to Biological Converter (DBC)](https://www.researchgate.net/publication/317237599_Digital-to-biological_converter_for_on-demand_production_of_biologics): a standalone machine that converts any input sequence to its chemical counterpart- DNA, RNA, proteins, viruses (1 â†’ many type converter)

[FlowJo](https://www.flowjo.com/): flow cytometry data analysis software (1996)

[Gibson Assembly](https://www.notion.so/Neural-Networks-1db5b5e4de3180fe97ced9dec4c6012a?pvs=21): method for joining/stitching DNA together 

[JCat](https://bio.tools/jcat): codon optimization tool 

[PUREexpress](https://www.neb.com/en-us/products/e6820-purexpress-disulfide-bond-enhancer?srsltid=AfmBOoqlhTxhSioBspUU522NwYX7J5Uuxu2yAWIIU8sJAc4VCkzvGtEt): cell free expression system

[Sleeping Beauty (SB100)](https://pmc.ncbi.nlm.nih.gov/articles/PMC4083583/): transposase system for stable circuit integration in mouse liver cancer models 

[SOLO Automated Liquid Handler](https://hudsonlabautomation.com/products/liquid-handlers/solo-automated-liquid-handler/): high resolution, compact robotic pipettor
